lib/usecases/applicationDetails.js

```diff
@@ -26,6 +26,10 @@ export default async ({ clientGeneratedId }) => {
       lrsgClosedBusinessesAmount: application.lrsg_closed_businesses_amount,
       lrsgSectorAmount: application.lrsg_sector_amount,
       lrsgOpenAmount: application.lrsg_open_amount,
+      lrsgClosedBusinessesPaymentExported:
+        application.lrsg_closed_businesses_amount,
```

Comment:
We want the export here, not the actual amount.

```suggestion
        application.lrsg_closed_businesses_payment_exported,
```